script/hassfest/dependencies.py

```diff
@@ -122,6 +122,7 @@ def visit_Attribute(self, node):
     "cover",
     "device_tracker",
     "fan",
+    "humidifier",
```

Comment:
Why do we change this in this PR?

Reply:
We are not actually changing it as the change is already there. I've reverted it to remove the confusion.